datacapture/src/test/java/com/google/android/fhir/datacapture/QuestionnaireViewModelTest.kt

```diff
@@ -4295,6 +4295,375 @@ class QuestionnaireViewModelTest {
     assertThat(enabledDisplayItems[0].text).isEqualTo("Text when yes is selected")
   }
 
+  @Test
+  fun `repeatedGroupItem response contains enabled items`() {
```

Comment:
1) this function needs to be moved up. under the section called "Questionnaire Response"

```suggestion
  fun `getQuestionnaireResponse() should return enabled repeated groups`() {
```

Reply:
done.